pyproject.toml

```diff
@@ -4,14 +4,13 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "taipy"
-version = "0.0.0"  # will be dynamically set
 description = "A 360° open-source platform from Python pilots to production-ready web apps."
-readme = "package_desc.md"
+readme = "README.md"
```

Comment:
We want to keep the `package_desc.md` here.

We differentiate the GitHub README from the package description file (PyPi description) used as project metadata. The README is used as GitHub's landing page and includes some marketing considerations not related to the Taipy package.